minecraft/getting-started.md

```diff
@@ -0,0 +1,9 @@
+---
+tag: "getting-started-paper"
+alias: ["getting-started-spigot", "getting-started-plugins"]
+---
+
+- Die [neue Java](<https://www.youtube.com/playlist?list=PLry1c-adUOIF0qnN6OK19L60CK2RXNAm0>) Serie befolgen und das was beigebracht wird lernen, [Hyperskill](<https://hyperskill.org/tracks/8?category=2>) hilft auch viel dabei.
+- [Paper MC downloaden](https://papermc.io/downloads), und [docs](<https://docs.papermc.io/paper/dev/getting-started>) lesen.
+- Projekt mithilfe von [IntelliJ](<https://www.jetbrains.com/idea/>) und [mit einem Gradle Guide von Chojo](<https://chojo.dev/blog/2023/08/04/minecraft-development-with-gradle-basic-and-advanced-a-guide/>) (es gibt auch eine Deutsche Version) erstellen.
+- Fehlendes Wissen welches nicht in den Paper MC docs aufgegriffen wird mit [dem Spigot Wiki](<https://www.spigotmc.org/wiki/spigot/>) und [Paper MC Javadocs](<https://jd.papermc.io/paper/1.20/>) auffüllen.
```

Comment:
```suggestion
- Die [neue Java](<https://www.youtube.com/playlist?list=PLry1c-adUOIF0qnN6OK19L60CK2RXNAm0>) Serie von BiVieh befolgen und das was beigebracht wird lernen. [Hyperskill](<https://hyperskill.org/tracks/8?category=2>) hilft auch viel dabei.
- [Paper downloaden](https://papermc.io/downloads) und [Paper Dokumentation](<https://docs.papermc.io/paper/dev/getting-started>) lesen.
- Projekt mithilfe von [IntelliJ](<https://www.jetbrains.com/idea/>) und [mit einem Gradle Guide von Chojo](<https://chojo.dev/blog/2023/08/04/minecraft-development-with-gradle-basic-and-advanced-a-guide/>) (es gibt auch eine Deutsche Version) erstellen.
- Wissen mit [dem Spigot Wiki](<https://www.spigotmc.org/wiki/spigot/>) und den [Paper Javadocs](<https://jd.papermc.io/paper/1.20/>) aufbessern.
```

Reply:
Took so long I reviewed the outdated revision, here are the remaining changes I proposed